Studio-Lovelies/GG-JointJustice-Unity
# Replaced included default player actions with custom player actions

## Summary
This PR removes all references in our project to the `DefaultPlayerActions` that are included as an example inside the `Input System` package and instead uses the `PlayerActions` instance at `/Assets/Scripts/Input/Control.cs` were using for the rest of the game already.

## Before/after screenshots and/or animated gif
### Unused UI input system actions
Unity's UI Input system consists of some behaviors we're currently not using. As we're not using them, I've deliberately left them unassigned:
- Middle Click
- Right Click
- Scroll Wheel
- Tracked Position and Orientation (not to be confused with the mouse position called `Point`, that's assigned; think a phone's gyro sensor or a Wiimote)

![image](https://user-images.githubusercontent.com/1689033/150483741-3c28a3c9-72ec-4a43-9909-d29cab1f6d0c.png)

### `DirectionalButtons` vs. individual `Left`, `Right`, `Up`, `Down` actions
Our `Controls` had both `DirectionalButtons` **and** individual `Left`, `Right`, `Up`, `Down` actions. I've removed the individual actions as they a) seemed unused and b) can easily be accessed by listening to changes of the `DirectionalButtons` axis.  
It now looks like this:  
![image](https://user-images.githubusercontent.com/1689033/150483754-fa82ccf5-3990-4e60-98e3-10252a5f5323.png)

### `X` and `Enter`
`X` and `Enter` can now be used interchangeably. I haven't really seen the need to separate the two, but if we want to split "Dialogue actions" and "UI actions" later on, it's something we can always do.

## Testing instructions
1. Run any scene of the game
2. Make sure all actions previously executable are still executable; aside from `X` and `Enter` now able to be used interchangeably, no keybinds should have changed
3. Verify the EditMode and PlayMode tests still succeed

## Additional information

- `[x]` Changes UI
  - `X` and `Enter` key can now be used interchangeably 
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[ ]` Has associated resource:
  - Closes #139
